docs/libs/rlottie.rst

```diff
@@ -243,23 +248,23 @@ The Esp-box devkit meets this spec and
 https://github.com/espressif/esp-box is a great starting point to adding
 lottie animations.
 
-you'll need to enable:c:macro:`LV_USE_RLOTTIE` through idf.py menuconfig under
+You will need to enable :c:macro:`LV_USE_RLOTTIE` through **idf.py** menuconfig under
 LVGL component settings.
 
 Additional changes to make use of SPIRAM
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
```

Comment:
I just want to add a note to this section as a reminder to update it. The ESP-IDF allows for setting memory thresholds so that anything being allocated above a specific set size will get allocated in SPIRAM instead of SRAM.  It does this without having to use the espidf malloc function.